apps/web/src/modules/job/components/card/index.tsx

```diff
@@ -0,0 +1,51 @@
+import { Divider, Typography } from '@mui/material'
+import React from 'react'
+
+import Footer from './footer'
+import Header from './header'
+import ShootingDetail from './shootingDetail'
```

Comment:
Please move file and change this
```suggestion
import Footer from './components/Footer'
import Header from './components/Header'
import ShootingDetail from './components/ShootingDetail'
```